pyrcareworld/pyrcareworld/attributes/base_attr.py

```diff
@@ -2,39 +2,43 @@ class BaseAttr:
     """
     Base attribute class, which includes general functions such as
     object loading, deleting and transforming.
-    """
 
-    def __init__(self, env, id: int, data: dict = {}):
-        self.env = env
-        self.id = id
-        self.data = data
+    By default, messages received from Unity are expected to include
+    the following keys:
 
-    def parse_message(self, data: dict):
-        """
-        Parse messages. This function is called by internal function.
+        'name': The name of the object.
 
-        Returns:
```

Comment:
Another reason I'm proposing this refactor is that the `Returns' here confused me (since this function doesn't return)